src/app/pages/homepage/homepage.html

```diff
@@ -6,55 +6,84 @@ <h1 class="mat-h1">Angular Material</h1>
       <h2> Material Design components for Angular</h2>
     </div>
     <div class="docs-header-start">
-     <a mat-raised-button class="docs-button" routerLink="/guide/getting-started">Get started</a>
+      <a mat-raised-button class="docs-button" routerLink="/guide/getting-started">Get started</a>
     </div>
   </div>
 </header>
 
 <main class="docs-homepage-promo">
-  <div class="docs-homepage-row">
-    <div class="docs-homepage-promo-img">
-      <docs-svg-viewer src="../assets/img/homepage/sprintzerotoapp.svg"
-                       [scaleToContainer]="true"></docs-svg-viewer>
+  <div class="docs-homepage-row docs-introduction">
+    <div class="docs-homepage-row-column">
+      <h2>High quality</h2>
+      <p>
+        Internationalized and accessible components for everyone. Well tested to ensure performance and reliability.
+      </p>
+      <p>Straightforward APIs with consistent cross platform behaviour.</p>
     </div>
-    <div class="docs-homepage-promo-desc">
-      <h2>Sprint from Zero to App</h2>
-      <p>Hit the ground running with comprehensive, modern UI components that work across
-        the web, mobile, and desktop.</p>
-    </div>
-  </div>
-  <div class="docs-homepage-row docs-homepage-reverse-row">
-    <div class="docs-homepage-promo-img">
-      <docs-svg-viewer src="../assets/img/homepage/fastandconsistent.svg"
-                       [scaleToContainer]="true"></docs-svg-viewer>
+    <div class="docs-homepage-row-column">
+      <h2>Versatile</h2>
+      <p>
+        Provide tools that help developers build their own custom components with common interaction patterns.
+      </p>
+      <p>
+        Customizable within the bounds of the Material Design specification.
+      </p>
     </div>
-    <div class="docs-homepage-promo-desc">
-      <h2>Fast and Consistent</h2>
-      <p>Finely tuned performance, because every millisecond counts. Fully tested across
-        modern browsers.</p>
+    <div class="docs-homepage-row-column">
+      <h2>Frictionless</h2>
+      <p>
+        Built by the Angular team to integrate seamlessly with Angular.
+      </p>
+      <p>
+        Start from scratch or drop into your existing applications.
+      </p>
     </div>
   </div>
-  <div class="docs-homepage-row">
-    <div class="docs-homepage-promo-img">
-      <docs-svg-viewer src="../assets/img/homepage/versatile.svg"
-                       [scaleToContainer]="true"></docs-svg-viewer>
-    </div>
-    <div class="docs-homepage-promo-desc">
-      <h2>Versatile</h2>
-      <p>Themeable, for when you need to stay on brand or just have a favorite color.
-        Accessible and internationalized so that all users are welcome.</p>
-    </div>
+
+  <mat-divider></mat-divider>
+
+  <div class="docs-homepage-featured-components">
+    <h1>Featured components</h1>
```

Comment:
nit: can this be an h2? I think the jury is still out on whether pages should only have a single h1 tag